extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -208,7 +209,8 @@ class AmpLightbox extends AMP.BaseElement {
    * @private
    */
   scrollHandler_() {
-    const currentScrollTop = this.element./*OK*/scrollTop;
+    // If scroll top is 0, it's set to 1 to avoid scroll-freeze issue.
+    const currentScrollTop = this.element./*OK*/scrollTop || 1;
```

Comment:
I think we still need `this.element./*OK*/scrollTop = 1;` or  `this.element./*OK*/scrollTop = currentScrollTop ` to actually adjust the scrollTop. Could you double check in the simulator that when opening up the lightbox, the scroll position of the `amp-lightbox` is changed to 1? And that the top scroll freeze problem is gone?